apps/manga-reader/web/src/routes/authenticated.tsx

```diff
@@ -0,0 +1,32 @@
+import { SignedIn, SignedOut } from "@clerk/clerk-react";
+import { css } from "@SMLCorp/styled-system/css";
+import { Navigate, Outlet } from "@tanstack/react-router";
+
+import { Menu } from "../components/menu";
+import { loginRoute } from "./routes";
+
+const Auth = (): JSX.Element => (
+  <>
+    <SignedIn>
+      <main
+        className={css({
+          height: "90svh",
+        })}
```

Comment:
**Suggestion:** The unit 'svh' seems to be a typo. It should be 'vh' for viewport height.
```suggestion
        className={css({
          height: "90vh",
        })}
```